src/Plugins/ResizeMirror/README.md

```diff
@@ -0,0 +1,45 @@
+## ResizeMirror
+
+The resize mirror plugin changes the mirror element dimensions as you drag it over other draggable elements.
+It also appends the mirror element into draggable containers you drag into.
+Add transitions to the mirror element to animate the dimensions change.
+
+This plugin is not included by default, so make sure to import.
```

Comment:
I wanted this to be a little more explicit, but i'm not sure what the right term is. but something like: `This plugin is not included in the default Draggable bundle, so you'll need to import it separately.` I don't know whether it should be "bundle" or "package" or what

Reply:
Bundle is perfectly fine 👍 a package could potentially contain multiple bundles